### ISSUE TITLE
[Screen reader - Logic Apps - Workflow settings]: Screen reader does not announce label and tooltip information of info button.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6108]
### Describe the issue 

User Experience:

 Users who rely on screen reader for navigation will be impacted if screen reader does not announce label and tooltip information of the info button as users will not get complete information of the control and will be misguided.

Note: User credentials should NOT be included in the bug.

Repro Steps:

- Open [https://ms.portal.azure.com/](<https://ms.portal.azure.com/>) and login via valid credentials. 
- Navigate to the Left Nav bar and select 'More Services', search 'Logic Apps' and select it. Select any already created logic app. 
- Select 'Work Flow Settings' in the left navigation pane. 
- Navigate all the controls on the 'Work Flow Settings' blade. 
- Verify whether screen reader announces label and tooltip information of info button.. 

Actual Result:

 Screen reader does not announce label and tooltip information of info button.

 Observation:

 Screen reader only announces button when focus lands on the info control.

Expected Result:

 Screen reader should announce label and tooltip information of info button. Screen reader should announce the information like "select an integration account button integration account…" when focus lands on info button.

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*

**Comment by hartra344** [comment]
https://msazure.visualstudio.com/One/_workitems/edit/29400110

**Comment by hartra344** [comment]
@Eric-B-Wu can you take a look at this one? It looks like a biproduct of when we did an overhaul of info bubbles

**Comment by hartra344** [comment]
nevermind, I got it
